test/FormattedMessage.test.js

```diff
@@ -0,0 +1,23 @@
+import FormattedMessage from '../src/FormattedMessage';
+
+describe('<FormattedMessage /> rendering', () => {
+  const randomId = 'Random Id';
+  const values = {
+    valueToBeInjected: 'Random Value',
+  };
+
+  it('should have default value and alt props', () => {
+    expect(FormattedMessage.defaultProps.values).toBeDefined();
+    expect(FormattedMessage.defaultProps.alt).toBeDefined();
+  });
+
+  it('should render FormattedMessage', () => {
+    let wrapper = shallow(<FormattedMessage id={randomId} />);
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  it('FormattedMessage should have span element', () => {
+    let wrapper = mount(<FormattedMessage id={randomId} values={values} />);
+    expect(wrapper.find('span')).toHaveLength(1);
+  });
```

Comment:
While all these tests may be valid, it doesn't really mock the React.Context with the TextContext and check if the value is actually injected during runtime.

That's the real test for `<FormattedMessage />`

Reply:
This is taken care of in the TextProvider test case. So, didn't add a test case mocking React.Context again.